account_chart_update/wizard/wizard_chart_update_view.xml

```diff
@@ -87,6 +88,63 @@
                         </page>
                     </notebook>
                 </page>
+                <page string="Matching"
+                      attrs="{'invisible': [('update_tax', '=', False), ('update_account', '=', False), ('update_fiscal_position', '=', False)]}"
+                >
+                    <h3>
+                        <p>Here you can set the matching order.</p>
+                    </h3>
+                    <notebook>
+                        <page string="Taxes"
+                              name="page_matching_taxes"
+                              attrs="{'invisible': [('update_tax', '=', False)]}"
+                        >
+                            <field name="tax_matching_ids"
+                                   context="{'account_chart_update': True}"
```

Comment:
Why do you need this context?

Reply:
removed

Reply:
I'm seeing that this is used in https://github.com/OCA/account-financial-tools/blob/10.0/account_chart_update/models/ir_model_fields.py#L13, but I don't think you need to make the same trick in other place.